src/View/AbstractDirectoryView.vala

```diff
@@ -3463,7 +3472,8 @@ namespace Files {
                             /* Determine whether should activate on key release (unless pointer moved)*/
                             /* Only activate single files with unmodified button when not on blank unless double-clicked */
                             if (no_mods && one_or_less) {
-                                should_activate = (on_directory && !on_blank && !singleclick_select) || double_click_event;
+                                should_activate = (on_directory && !on_blank && !singleclick_select)
+                                    || double_click_event;
```

Comment:
You shouldn't make changes which are unrelated to the issue in hand - however tempting it may be! You can always submit a separate "clean up the code" PR which makes no substantive changes but improves the formatting. Its best to keep the size of diff as small as possible